widget/progressbar.go

```diff
@@ -136,7 +136,7 @@ func (p *ProgressBar) CreateRenderer() fyne.WidgetRenderer {
 		p.Max = 1.0
 	}
 
-	background := canvas.NewRectangle(theme.ShadowColor())
+	background := canvas.NewRectangle(progressBackgroundColor())
```

Comment:
This change is in `CreateRenderer`, is your `Send` UI recreated each time it is selected?

Reply:
Oh I see, I just initialized the background colour incorrectly